### ISSUE TITLE
KER Doesn't Recognize dV for Jet Engines

### ISSUE THREAD

**Issue by MInorInconvenience** [CYBUTEK/KerbalEngineer#38]
it seems tha the latest branch doesn't recognize that jet engines pull fuel equally even if not connected by fuel lines or connected directly. The dV readings in the SPH and in flight are wrong

**Comment by Gerry1135** [comment]
Yes, this is a known problem.  It has already been fixed in PR #34 but there are a few other issues that also need fixing before another release can be done.

**Comment by arekbulski** [comment]
How about closing this thread? Its already fixed.
